.github/workflow/sync-automation.yml

```diff
@@ -0,0 +1,45 @@
+# Generated by: wf-gen from tyk-ci
+# Generated on: Wed  9 Dec 14:16:13 IST 2020
+
+
+# This workflow will silently ignore FILES that are missing in master
+# Most of the time this is what we want, older repos may not have some files
+
+name: Sync automation
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/*
+      - integration/*
```

Comment:
Still I guess it should work with feature/* branches?

Reply:
These are the paths that the workflow will be triggered on. .`g/w/*` contains the workflows and `integration/*` contains some docker and terraform stuff needed to make the whole thing work.

Reply:
sync-automation is purely involved in syncing workflows to all the release branches, not directly related to the developer environments.